src/structure/options.ts

```diff
@@ -131,7 +131,7 @@ export class StructureOptions extends OptionsGroup {
             );
         }
         this.color.mode.validate = optionValidator(['linear', 'log', 'sqrt', 'inverse'], 'mode');
-        this.color.palette.validate = optionValidator(['Rwb', 'Roygb', 'Sinebow'], 'palette');
+        this.color.palette.validate = optionValidator(['bwr', 'Roygb', 'Sinebow'], 'palette');
```

Comment:
These should have similar casing. Either `bwr`, `roygb` and `sinebow` or `Bwr`, `Roygb` and `Sinebow`. I could also see a point for `BWR`, `ROYGB` and `sinebow` since the first two refer to colors names.

Reply:
Actually, there is a line in the code that Angelo was using to inverse max & min in the gradient. That's probably why it is BWR instead of RWB.

Reply:
agree. I'd vote for BWR, ROYGB and sinebow